sdk/transaction/src/sanitized.rs

```diff
@@ -338,14 +336,13 @@ impl SanitizedTransaction {
 mod tests {
     use {
         super::*,
-        crate::{
-            reserved_account_keys::ReservedAccountKeys,
-            signer::{keypair::Keypair, Signer},
-        },
+        solana_keypair::Keypair,
         solana_program::{
-            message::MessageHeader,
+            message::{MessageHeader, SimpleAddressLoader},
             vote::{self, state::Vote},
```

Comment:
Looks like we'll need to keep `solana-program` as a dev-dependency for this at least, until we move the vote client out